privatebin/templates/statefulset.yaml

```diff
@@ -15,6 +15,9 @@ spec:
     matchLabels:
       app.kubernetes.io/name: {{ include "privatebin.name" . }}
       app.kubernetes.io/instance: {{ .Release.Name }}
+  {{- if .Values.controller.serviceName }}
+  serviceName: {{ .Values.controller.serviceName | quote }}   
+  {{- end }}
```

Comment:
since a StatefulSet always requires serviceName we don't need a conditional
```suggestion
  serviceName: {{ include "privatebin.fullname" . }}
```

Reply:
Thanks for your review!